tests/test_data_collections.py

```diff
@@ -89,25 +110,37 @@ def test_attributes() -> None:
     assert data_collection.service_url is None
 
 
-def test_sentinel1_checks() -> None:
-    assert DataCollection.SENTINEL1_IW.is_sentinel1
-    assert not DataCollection.SENTINEL2_L1C.is_sentinel1
-
-    assert DataCollection.SENTINEL1_IW_ASC.contains_orbit_direction("ascending")
-    assert not DataCollection.SENTINEL1_IW_DES.contains_orbit_direction("ascending")
-
-    assert DataCollection.SENTINEL2_L2A.contains_orbit_direction("descending")
+@pytest.mark.parametrize(
+    "test_collection, expaected",
+    [
+        (DataCollection.SENTINEL2_L1C, False),
+        (DataCollection.SENTINEL1_EW, True),
+        (DataCollection.LANDSAT_TM_L1, False),
+    ],
+)
+def test_is_sentinel1(test_collection: DataCollection, expaected: bool) -> None:
+    assert test_collection.is_sentinel1 == expaected
+
+
+@pytest.mark.parametrize(
+    "collection, direction, expaected",
+    [
+        ("SENTINEL1_IW_ASC", "ascending", True),
+        ("SENTINEL1_IW_ASC", "descending", False),
+        ("SENTINEL1_IW_DES", "ascending", False),
+        ("SENTINEL2_L2A", "descending", True),
+        ("SENTINEL2_L2A", "ascending", True),
+    ],
+)
+def test_contains_orbit_direction(collection: str, direction: str, expaected: bool) -> None:
+    data_collection = getattr(DataCollection, collection)
+    assert data_collection.contains_orbit_direction(direction) == expaected
 
 
 def test_get_available_collections() -> None:
     collections = DataCollection.get_available_collections()
-    assert helper_check_collection_list(collections)
-
-
-def helper_check_collection_list(collection_list: Any) -> bool:
-    is_list = isinstance(collection_list, list)
-    contains_collections = all(isinstance(data_collection, DataCollection) for data_collection in collection_list)
-    return is_list and contains_collections
+    assert len(collections) >= 28
```

Comment:
why is it 28? if we define a new one, will it be 29? if the user defines a new one, will it be then 30? do byocs count? do batch collections?

Reply:
Removed ... 28 are set on init, answer yes, yes, yes and yes.

Reply:
but how do you know that is the case? Maybe you should test it :P

Reply:
Tested.